wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -468,6 +471,12 @@ int atmel_init(void)
     int ret = 0;
 
 #if defined(WOLFSSL_ATECC508A) || defined(WOLFSSL_ATECC608A)
+
+/*Harmony3 will generate configuration based on user inputs*/
+#ifdef MICROCHIP_MPLAB_HARMONY_3
+    extern ATCAIfaceCfg atecc608_0_init_data;
```

Comment:
I suspect this will not be an extern available for `WOLFSSL_ATECC508A`. Might be good to wrap this and the one at line 505 with `WOLFSSL_ATECC608A`.